File: src/ResumeTailor.Application/Resumes/Models/CompanyGeneratedResume.cs
namespace ResumeTailor.Application.Resumes.Models;

public sealed record CompanyGeneratedResume(
    string PersonName,
    string Profession,
    string Email,
    string PhoneNumber,
    string? College,
    string? Degree,
    string? Major,
    string? CollegeStatus,
    string? PersonalSite1,
    string? PersonalSite2,
    string? PersonalSite3,
    List<string> Skills,
    List<GeneratedCompany> Companies);

File: src/ResumeTailor.Client/Web/src/models/resume/CompanyGeneratedResume.ts
import type { GeneratedCompany } from "./GeneratedCompany";
import type { Project } from "./Project";

export type CompanyGeneratedResume = {
  personName: string;
  profession: string;
  location: string;
  phoneNumber: string;
  college?: string;
  degree?: string;
  major?: string;
  collegeStatus?: string;
  personalSite1?: string;
  personalSite2?: string;
  personalSite3?: string;
  skills: string[];
  experience?: GeneratedCompany[];
  projects?: Project[];
};
